src/runtime_src/core/edge/drm/zocl/zocl_ctrl_ert.c

```diff
@@ -753,7 +762,15 @@ static int zert_probe(struct platform_device *pdev)
 	if (!zert->zce_ctrl_xgq_hdl)
 		zert_err(zert, "failed to initialize CTRL XGQ");
 
-	return 0;
+	zert_init_cus(zert);
+	zert->zce_max_cu_size = CTRL_XGQ_CU_SLOT_SIZE;
```

Comment:
do you still need this member variable - zce_max_cu_size?

Reply:
Fixed